core/src/scss/components/date-stacked/_date-stacked--no-background.scss

```diff
@@ -0,0 +1,11 @@
+@charset "UTF-8";
+
+//
+// Date Stacked No Background
+//
+
+.su-date-stacked--no-background {
+  @include padding(0);
+  background: none;
+  color: $color-black;
```

Comment:
If we're going to merge master into this branch, we could get the colors from the color map. We could leave it as variables for now also.

Reply:
Looks great - thanks for the change!